### DISCUSSION TITLE
Admin/Backend area for user control?

### DISCUSSION THREAD

**Discussion by FelixBuehler** [lukevella/rallly#1435] [Q&A] [answered]
Hey everybody,

I'm asking myself, if this app has some sort of backend area, where i can see all polls and all registered users. Or if there is some kind of admin account?

Thanks!

**Comment by lukevella** [comment, marked as answer]
There isn't right now. You'd need to use direct access to the database. Would be interesting to build an interface for admins at some point but it's not currently planned.

**Reply by FelixBuehler** [reply]
Thanks for the fast reply. Would be awesome to have such a feature.
